### PR TITLE
built files???

### PR DESCRIPTION
**Pull request by amirbey**

### CODE REVIEW

File: publishing/s3publisher.py [hunk 1]
```diff
@@ -193,7 +193,8 @@ def publish_to_s3(directory, base_url, site_prefix, bucket, cache_control,
         if not local_objects_by_filename.get(filename)
     ]
 
-    if len(new_objects) == 0 and len(replacement_objects) == 0:
+    if (len(new_objects) == 0 and len(replacement_objects) <= 1 and
+            len(local_files) <= 1):
```
Comment (davemcorwin) [inline publishing/s3publisher.py]: we could probably just use `len(local_files) <= 1`...

Comment (amirbey) [reply publishing/s3publisher.py]: agreed ... however, its a problem for our tests :(

Comment (davemcorwin) [reply publishing/s3publisher.py]: Ideally, our tests would reflect what we want the logic to be, right now we're not testing this at all, maybe we should?

**Comment by amirbey** [comment]
ok. i agree  .... my intention is that this a temporary solution while we work uncover the underlying issue that this claus is blocking. in the immediate, i  think the cost:benefit of rewriting the tests for a sole clause for this condition is not worth the effort to provide immediate relief.

i think we can also spend some time to beef up tests overall

**Pull request merged**